README.md

```diff
@@ -733,6 +734,64 @@ More .NET Podcasts can be found on [The Sound of .NET](https://thesoundof.net/)
   * **Frequency**: Approximately once a month
   * **Runtime**: 20 - 70 mins, regularly ~50 mins
 
+## InfoSec
```

Comment:
There is Security section, can we merge these 2 list plz

Reply:
of course. i was pretty sure i checked in the beginning, if it was already there.. ^^